client/src/luigi-client.js

```diff
@@ -410,6 +419,30 @@ const LuigiClient = {
           { msg: 'luigi.set-page-dirty', dirty: isDirty },
           '*'
         );
+      },
+      /**
+       * Shows a confirmation modal.
+       * @param {Object} content the content of the confirmation modal
+       * @param {string} content.header the content of the modal header
+       * @param {string} content.body the content of the modal body
+       * @param {string} content.buttonConfirm the label for the modal confirm button
+       * @param {string} content.buttonDismiss the label for the modal dismiss button
+       * @returns {promise} A promise which is resolved when accepting confirmation modal and rejected when dismissing it.
```

Comment:
```suggestion
       * @returns  a {promise} which is resolved when accepting the confirmation modal and rejected when dismissing it.
```

Reply:
This is not correct regarding documentation. First comes the type of the returned value and the the additional info. I will just add the "the"

Reply:
Ok :) I was just looking at the entire document, and under `has Back` I've found:
`Returns boolean indicating if there is a preserved view you can return to.` that's why I wanted to adjust the other part of the doc. Should we change this one as well, or is it a different case?

Also let's then write `a promise`  - lower case, to make it consistent :)

Reply:
The piece of documentation you mentioned is:
```
       * @returns {boolean} indicating if there is a preserved view you can return to.
```
I will replace the line we are talking about with:
```
       * @returns {promise} which is resolved when accepting the confirmation modal and rejected when dismissing it.
```
This makes it consistent imho. Is that ok for you?